test/unit/specs/common/cd-special-req-component.spec.js

```diff
@@ -0,0 +1,125 @@
+import vueUnitHelper from 'vue-unit-helper';
+import SpecialReqComponent from '@/common/cd-special-req-component';
+
+describe('Special Requirement Component', () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('methods', () => {
+    describe('methods.onBlur()', () => {
+      it('should emit a blur event after 50ms and assign the timeout to blurTimeout', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(SpecialReqComponent);
+        sandbox.stub(window, 'setTimeout').callsFake((cb) => {
+          cb();
+          return 'foo';
+        });
+        const emitStub = sandbox.stub();
+        vm.$emit = emitStub;
+
+        // ACT
+        vm.onBlur();
+
+        // ASSERT
+        expect(window.setTimeout).to.have.been.calledOnce;
+        expect(window.setTimeout).to.have.been.calledWith(sinon.match.func, 50);
+        expect(vm.$emit).to.have.been.calledOnce;
+        expect(vm.$emit).to.have.been.calledWith('blur');
+        expect(vm.blurTimeout).to.equal('foo');
+      });
+    });
+
+    describe('methods.onFocus()', () => {
+      it('should clear any existing blur timeout', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(SpecialReqComponent);
+        sandbox.stub(window, 'clearTimeout');
+        vm.blurTimeout = 'foo';
+
+        // ACT
+        vm.onFocus();
+
+        // ASSERT
+        expect(window.clearTimeout).to.have.been.calledOnce;
+        expect(window.clearTimeout).to.have.been.calledWith('foo');
+      });
+    });
+  });
+
+  describe('computed', () => {
+    describe('computed.hasSpecialReq', () => {
```

Comment:
hasSpecialReq is not computed

Reply:
sorry didnt get to tests yet, was just about to fix em